controller/work_item_comments_blackbox_test.go

```diff
@@ -125,6 +126,37 @@ func assertWorkItemComment(t *testing.T, c *app.Comment, expectedBody string, ex
 	assert.NotNil(t, c.Relationships.Creator.Data.ID)
 }
 
+func (rest *TestCommentREST) TestSuccessCreateSingleCommentWithParentComment() {
+	// given
+	wi := rest.createDefaultWorkItem()
+
+	// create parent
+	markup := rendering.SystemMarkupMarkdown
+	p := rest.newCreateWorkItemCommentsPayload("Test", &markup)
+	svc, ctrl := rest.SecuredController()
+	_, c := test.CreateWorkItemCommentsOK(rest.T(), svc.Context, svc, ctrl, wi.ID, p)
+	// then
+	assertComment(rest.T(), c.Data, rest.testIdentity, "Test", markup)
+
+	// create child
+	parentID := c.Data.ID.String()
+	child := rest.newCreateWorkItemCommentsPayload("Test Child", &markup)
+	child.Data.Relationships = &app.CreateCommentRelations{
+		ParentComment: &app.RelationGeneric{
+			Data: &app.GenericData{
+				Type: ptr.String("comments"),
```

Comment:
Looks like there are few repetitive usages of the string literal: `"comments"`. Similar to other type constants, we can create a constant named `APIStringTypeComments` inside the `comments` package and use it in `controller` and `controller_test` packages.

Reply:
Please grep for `"comments"` to see other occurrences of the string literal.

Reply:
Done. I did not change these on the tests, though as the tests should check if the correct type is used. If the tests would use the constant from the code, a wrong constant would not break the code. Also, the constant is repeated in the test. This is also intended as the tests are supposed to break early.